apps/web/env.js

```diff
@@ -1,30 +1,30 @@
 /* eslint-disable @typescript-eslint/no-var-requires */
 require('dotenv').config();
 
-const port = process.env.PORT || 3030;
```

Comment:
@Anishali2 why you commented this whole file code out? Do you know why that file was needed etc?

Reply:
No, please uncomment it now and this logs are needed because they inform user on which port it's running etc!

Reply:
Ok Sir give me a sec